libs/api/domains/regulations-admin/src/lib/graphql/dto/downloadRegulation.input.ts

```diff
@@ -0,0 +1,9 @@
+import { Field, InputType } from '@nestjs/graphql'
+import { IsString } from 'class-validator'
+
+@InputType()
+export class DownloadRegulationInput {
+  @Field()
+  @IsString()
+  regulationId!: string
```

Comment:
```suggestion
  draftId!: string
```

Calling it regulationId is confusing.